### ISSUE TITLE
Plugin won't build with recent node

### ISSUE THREAD

**Issue by jxjj** [UMN-LATIS/elevator-wordpress#16]
I couldn't get the plugin to build using node 18 or 20. It looks like `cgb-scripts` is defunct. `@wordpress/create-block` might be an alternative.

**Comment by cmcfadden** [comment]
Yeah - not surprising. I'm not sure anyone is using this plugin right now - I'll ask around. Unless Matt just totally blows up Wordpress before that ...
